### PR TITLE
Indicate in configuration which admin monitors are security-related

### PR DESCRIPTION
**Pull request by daniel-beck**
Amends https://github.com/jenkinsci/jenkins/pull/5015 to add an indicator in the admin monitor configuration for those that are in the 'Security' category.

Suggestions for improving the design are welcome.

### Screenshot

> Screenshot 2021-03-05 at 14 00 26

### Proposed changelog entries

* Add indicator for security-related entries in the global administrative monitors configuration.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

File: core/src/main/resources/jenkins/management/AdministrativeMonitorsConfiguration/config.groovy
Comment (timja) [inline core/src/main/resources/jenkins/management/AdministrativeMonitorsConfiguration/config.groovy:45]: could we add a css class for this?

Comment (daniel-beck) [reply core/src/main/resources/jenkins/management/AdministrativeMonitorsConfiguration/config.groovy:45]: 🤷 sure

**Review by MRamonLeon — APPROVED** [review]
Fine with the goal. Agree on using a css class

File: core/src/main/resources/jenkins/management/AdministrativeMonitorsConfiguration/config.groovy
Comment (fqueiruga) [reply core/src/main/resources/jenkins/management/AdministrativeMonitorsConfiguration/config.groovy:45]: Makes sense, let me know if you would like a hand like we did with the plugin manager badges.

Comment (daniel-beck) [reply core/src/main/resources/jenkins/management/AdministrativeMonitorsConfiguration/config.groovy:45]: @fqueiruga Sure, any help to not make this look ugly is appreciated.

Comment (fqueiruga) [reply core/src/main/resources/jenkins/management/AdministrativeMonitorsConfiguration/config.groovy:45]: Sure, just let me know once the PR is ready 👍

**Review by oleg-nenashev — APPROVED** [review]
It would be great to have a more generic way to do it in the future so that we could support more labels. And yes, moving the layout to CSS so that it can be overridden in themes. No concerns feature wise, it is a good improvement

**Comment by oleg-nenashev** [comment]
We may merge it in 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process

**Comment by timja** [comment]
There’s agreement in the PR to move this to using a css class, let’s do that first

**Comment by oleg-nenashev** [comment]
Agreed, sorry for the confusion

**Comment by fqueiruga** [comment]
Yes, I'm trying to find some spare time to do it

**Comment by daniel-beck** [comment]
@fqueiruga Ping?

File: core/src/main/resources/jenkins/management/AdministrativeMonitorsConfiguration/config.groovy
Comment (daniel-beck) [reply core/src/main/resources/jenkins/management/AdministrativeMonitorsConfiguration/config.groovy]: Fixed.

**Review by fqueiruga — APPROVED** [review]

**Review by timja — APPROVED** [review]
tested locally in both regular and dark themes looks good

image

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Pull request merged**
